### ISSUE TITLE
Does vmine support asynchronous API?

### ISSUE THREAD

**Issue by xiexin0606** [kisli/vmime#266]
Hi, does vmine support asynchronous API?

**Comment by RoumedG** [comment]
No. If you need some concurency, you have to implement it by yourself.
It will be useful to see some classes at src/vmime/platforms. Good luck.
